webapp/i18n/nl.json

```diff
@@ -1116,7 +1116,7 @@
   "channel_modal.channel": "Kanaal",
   "channel_modal.createNew": "Maak nieuw ",
   "channel_modal.descriptionHelp": "Beschrijf hoe deze {term} gebruikt moet worden.",
-  "channel_modal.displayNameError": "Dit is een verplicht veld",
+  "channel_modal.displayNameError": "Kanaalnaam moet 2 of meer kleine alfanumerieke karakters bevatten",
```

Comment:
Please don't update non English translation files. I believe it breaks the translation server. If you want to provide translation you can see the docs in the header of this channel: https://pre-release.mattermost.com/core/channels/localization  Also feel free to ask questions there!